src/pkg/runner/runner.go

```diff
@@ -397,6 +379,11 @@ func (r *Runner) performAction(action types.Action) error {
 			return err
 		}
 
+		// template the withs with variables
```

Comment:
this is a new "feat"? assuming this means we weren't allowing  variables w/ "withs"?

Reply:
yes!